Dockerfile.backwards

```diff
@@ -7,8 +7,7 @@ RUN echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/base.yaml" > /etc/ros/r
 COPY . /opt/package
 WORKDIR /opt/package
 
-RUN apt-get install curl
-RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+RUN apt update || apt install -y curl && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

Comment:
The `||` in `apt update || apt install` will run `apt install` if `apt update fails. I think you meant `&&` which would short-circuit on `apt update` failure.